src/components/Dashboard/Utilization/UtilizationItem.js

```diff
@@ -49,13 +51,21 @@ export class UtilizationItem extends React.PureComponent {
     if (isLoading) {
       chart = <LoadingComponent />;
     } else if (data) {
-      const chartData = {
-        columns: [[unit, ...data.map(val => val.toFixed(1))]],
-        unload: true,
-        type: 'area',
-      };
+      const chartData = data.map((val, index) => ({ x: index, y: Number(val.toFixed(1)) }));
       actual = `${Math.round(data[data.length - 1])} ${unit}`;
-      chart = <SparklineChart id={id} data={chartData} axis={axis} unloadBeforeLoad />;
+      chart = (
+        <Chart
+          id={id}
+          height={60}
+          width={300}
+          containerComponent={<ChartVoronoiContainer labels={datum => `${datum.y} ${unit}`} />}
```

Comment:
the label is huge! Can something be done about that? On the other hand the pies' labels are quite small.

Reply:
I tried to make the tooltips similar, but it will need more work in the future